src/components/actionButton/styles.ts

```diff
@@ -0,0 +1,37 @@
+import ThemeContext from '@contexts/ThemeContext';
+import { useContext } from 'react';
+import { StyleSheet } from 'react-native';
+
+const styles = () => {
+  const {
+    theme: { actionButton: theme },
+  } = useContext(ThemeContext);
```

Comment:
explain the theme flow

Reply:
So i have define some type for theme colors

`type ThemeMode = 'light' | 'dark';

  interface ThemeColors {
    statusBar: string;
    screenBG: string;
    defaultIcon: string;
    }
    
    interface Theme {
    light: ThemeColors;
    dark: ThemeColors;
  }    
    `